Mobile

DmitriiTsy opened the pull request.

DmitriiTsy committed:
mobile version

DmitriiTsy committed:
first mobile

DmitriiTsy commented:
I fixed some major issues such as naming, tags (I changed it from  the divs to regular - "h, header, span etc") Should be helpful for semantic optimisation

There are some minor issues with spaces that I will fix with your new feedback

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    gap: 40px
+}
+
+.header__glass{
```
missing space

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    position: absolute;
+}
+
+.header__small-container{
```
what is an element and what is a modificator here? small sounds like a modificator? doesn't it?

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+}
+
+.header__small-container{
+    display:flex;
```
space missed

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    padding: 100px 0px 100px 0px;
+}
+
+.let-ride h1 {
```
tag selector, avoid this

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    font-weight: 550;
+}
+
+.let-ride span {
```
another tag selector

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    padding: 200px 0px 0px 0px;
+}
+
+.drive-with span {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    width: 100%;
+} 
+
+.drive-with p{
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    flex-direction: column;
+}
+
+.drive-with__text-section p {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    font-weight: 300;
+}
+
+.drive-with__text-section strong {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    align-items: center;
+}
+
+.drive-with__how-driver-pay-container span{
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    align-items: center;
+}
+
+.drive-with__feedback p {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    line-height: 1.5;
+}
+
+.drive-with__feedback span {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    gap: 30px;
+}
+
+.ride span {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    width: 100%;
+}
+
+.ride h2 {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    width: 100%;
+}
+
+.ride p {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    width: 100%;
+}
+
+.ride__list strong {
```
you know what to do with that :D

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    height: 100px;
+}
+
+.ride__cards p{
```
the same comment for the rest tag selectors

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+    display:flex;
+    flex-direction: column;
+    text-align: left;
+    background-color:#DCDCFF;
```
space is missing between rule name and rule value, make sure you fix it everywhere

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,771 @@
+}
+
+.estimation__textareas{
+    display:flex;
```
you can also fix this by installing Stylelint plugin to VS Code

anatoliy8493 commented on main.html:
```diff
@@ -1,9 +1,367 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Page Title</title>
```
let's add normalize.css here as we discussed

ghost commented:
> I fixed some major issues such as naming, tags (I changed it from the divs to regular - "h, header, span etc") Should be helpful for semantic optimisation
> 
> There are some minor issues with spaces that I will fix with your new feedback

let's fix all minor issues before sending a PR, it's very important for reviewers to be able to review 'prepared' PRs and don't spend extra time on this. Mind adding a screenshot? for the long websites, you can use https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl

ghost commented:
I noticed that you used `first` word for repo naming, let's avoid numbers for naming,`lyft-main-page` or `lyft-main` would be better (do not change anything, just a comment for the future)

DmitriiTsy commented:
PDF Version  [Lyft__mobile.pdf](https://github.com/DmitriiTsy/Lyft_first/files/8882382/Lyft__mobile.pdf)

Mobile Version ![Lyft__mobile](https://user-images.githubusercontent.com/101737790/173153564-db72095e-9b74-4f05-8e3f-e8653237a129.png)

Hi!, sure. This is the full screenshot of a new version with fixes:

1. I deleted all tags (p, span, etc)  in CSS and added classes;
2. Fixed all white spaces in the CSS;
3. Fixed more issues regarding page (spaces between elements, font sizes, etc)
4. I added the newest CSS normalize at the start of the page.  

@anatoliy8493

DmitriiTsy committed:
minor_fixes

DmitriiTsy committed:
final_fixes

DmitriiTsy commented:
I also have one specific question :) Is it important to use <footer>, <header> and etc. instead of "div" everywhere on the page? If yes, I will add them. 

The reason I asking is that I only added semantic text tags (p, span, h1, h2 and etc) but not all semantic tags

DmitriiTsy committed:
normaliza + semantic tags fix

DmitriiTsy commented:
Hi, @anatoliy8493 Added a new commit ("Normalize and semantic tags"): 

- Added normalize.css instead of reset.css that was before;
- Added more semantic tags;

* Attached w3 HTML validator analysis (errors in HTML tags and etc.) 

![Screen Shot 2022-06-13 at 15 07 08](https://user-images.githubusercontent.com/101737790/173453216-3bcc2cff-c889-49f3-bcca-6e0158bcf41d.png)

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,789 @@
+html,
+body {
+    font-family: sans-serif;
+    margin: 0px;
```
do we still need this after normilize.css was included?

anatoliy8493 commented on main.css:
```diff
@@ -0,0 +1,789 @@
+    align-items: center;
+}
+
+.drive-with__bold {
```
mixed up element and modificator, read https://en.bem.info/methodology/naming-convention/

anatoliy8493 commented on main.html:
```diff
@@ -1,9 +1,356 @@
+<meta charset="UTF-8">
+<title>Page Title</title>
+<link rel="stylesheet" href="main.css">
+<link rel="stylesheet" href="normalize.css">
```
normalize should be the 1st

anatoliy8493 commented on normalize.css:
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
```
can we load it from CDN and avoid storing at the repo?

anatoliy8493 reviewed (approved):
LGTM once we fix the last comments

DmitriiTsy commented:
Hi

I added     <meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0,
user-scalable=0' name='viewport' />

To my HTML in order to create responsive content on the page, depending on the device width. But after that, all of my previous work with layouts crashed.

Is it a best practice to use these meta tags all the time?  @anatoliy8493

DmitriiTsy commented:
![Screen Shot 2022-06-15 at 12 24 58](https://user-images.githubusercontent.com/101737790/173908180-c320a7e3-0038-441b-a8ee-c54d63703ff4.png)

DmitriiTsy commented:
![screencapture-192-168-1-9-56526-main-html-2022-06-16-15_26_27](https://user-images.githubusercontent.com/101737790/174188781-530cf29a-b403-427c-b1cd-5a03288a7452.png)


Apple Ipad Mini

DmitriiTsy commented:
![screencapture-192-168-1-9-56526-main-html-2022-06-16-15_28_32](https://user-images.githubusercontent.com/101737790/174188892-56d4d8e6-cbdc-40c4-a6ef-68493a4a9049.png)

Tablet (wide) - Nest Hub Max

DmitriiTsy commented:
![screencapture-192-168-1-9-56526-main-html-2022-06-16-15_29_52](https://user-images.githubusercontent.com/101737790/174189018-dbcfe203-6a5b-4dc6-83ef-9af8909282a1.png)

MacBook  13" (DeskTop)

DmitriiTsy commented:
Hi! 

I added 2 tablet versions and 1 desktop.  (Didn't create a cards section - because it depends on the flow and js as I understand right) 

@anatoliy8493 

What do you think?
